pyproject.toml

```diff
@@ -419,5 +419,6 @@ dev-dependencies = [
     "pytest>=8",  # Pin to keep recent
     "python-dotenv",
     "refurb>=2",  # Pin to keep recent
+    "sentence-transformers",  # for unit tests
```

Comment:
Let's move this to an extra, and then have `paper-qa[datasets,ldp,typing,zotero,NEWEXTRA]` above